src/tests/ftest/pool/multi_server_create_delete_test.py

```diff
@@ -63,7 +63,7 @@ def setUp(self):
         self.agent_sessions = agent_utils.run_agent(self.basepath,
                                                     self.hostlist_servers)
         server_utils.run_server(self, self.hostfile, server_group)
-        self.dmg = self.basepath + '/install/bin/dmg'
+        self.dmg = self.basepath + '/install/bin/dmg_old'
```

Comment:
Seems like this could be updated to use the new tool, ideally. Could be done as a separate PR in preparation for removing the old tool though.

Reply:
yes that is something that should be done, although not necessarily before landing this PR @amjustin13 @johannlombardi